### ISSUE TITLE
Build Error building the latest source. DiscordBridge.cs line 117. Missing ;

### ISSUE THREAD

**Issue by TheNightRider12** [nint8835/iTunesRichPresence#26]
```
Severity	Code	Description	Project	File	Line	Suppression State
Error	CS1002	; expected	iTunesRichPresence-Rewrite	D:\Desktop_Stuff\Github_Clones\C#\iTunesRichPresense\iTunesRichPresence\DiscordBridge.cs	117	Active
```
Here
```
            catch (EntryPointNotFoundException) {
                var newPresence = new DiscordRpc.RichPresence {
                    largeImageKey = "itunes_logo_big",
                    details = "No song playing",
                    state = "Re-install iTunesRichPresence to clear this message"
                }
            }
```
Adding a semi colon after the ending curly bracket let's the project build successfully.

**Comment by nint8835** [comment]
As you can tell, I put extensive testing into this fix. I'd notice the error when I went to build a patch, but thanks for pointing it out to me.

**Comment by nint8835** [comment]
Resolved in 06cfd81
